docs/source/modules/submission.rst

```diff
@@ -111,15 +110,15 @@ aptly named dev_score_models.sh. That script is what is run on Jenkins's dev env
 successfully, the code can be merged to master and will be run "in production".
 
 Scoring Process Description
-#################
+###########################
 For scoring submitted files, we install the Brain-Score framework on Opemnind and run the scoring process. There are
 two types of submissions possible:
-   - First time submissions, submitting a zip file with new models to score.
-   - Resubmission of already scored models, which should be scored on updated/new benchmarks.
+- First time submissions, submitting a zip file with new models to score.
+- Resubmission of already scored models, which should be scored on updated/new benchmarks.
```

Comment:
@mike-ferguson Aren't existing models automatically scored on any updated/new benchmarks?

Reply:
It used to be that they have to be manually triggered (benchmark PR is merged, a single model is scored, it shows up on site, admin scores all models on them) @mschrimpf is this still the case?